tests/e2e/constants/BASE_TEST_CONSTANTS.ts

```diff
@@ -53,15 +54,22 @@ export const BASE_TEST_CONSTANTS: {
 	 */
 	TEST_ENVIRONMENT: process.env.TEST_ENVIRONMENT || '',
 
+	/**
+	 * openshift project or k8s namespace which is used by test
+	 */
+	TEST_NAMESPACE: process.env.TEST_NAMESPACE || '',
+
 	/**
 	 * application name (DevSpaces or Che)
 	 */
 	TESTING_APPLICATION_NAME: (): string => {
-		return BASE_TEST_CONSTANTS.TS_SELENIUM_BASE_URL.includes('devspaces')
-			? 'devspaces'
+		return BASE_TEST_CONSTANTS.TEST_NAMESPACE.length > 0
+			? BASE_TEST_CONSTANTS.TEST_NAMESPACE
```

Comment:
@musienko-maxim : this multi line statement is really hard to read, which could lead to mistakes in future.
What do you think about using if-statements?